r/R/feather.R

```diff
@@ -144,17 +144,20 @@ write_feather <- function(x,
 #' }
 read_feather <- function(file, col_select = NULL, as_data_frame = TRUE, ...) {
   if (!inherits(file, "RandomAccessFile")) {
-    file <- make_readable_file(file)
+    file <- make_readable_file(file, ...)
```

Comment:
Thanks for moving these `...`. You should also update the docs above.

Reply:
not sure how to refer to an internal function in Rd linking, so I just changed to "additional parameters." for now

Reply:
old:
```
csv.R:#' - `...` additional parameters.
feather.R:#' @param ... additional parameters, passed to the internal function
```

new:
```
csv.R:#' - `...` additional parameters.
feather.R:#' @param ... additional parameters.
```

Reply:
`FeatherReader` is an exported object with a docs entry, so this link will work fine.

Reply:
so, I just keep 
```
feather.R:#' @param ... additional parameters, passed to [FeatherReader$create()][FeatherReader]
```
or I move to
```
#' @param ... additional parameters, passed to the internal function
#' `make_readable_file()`
```

Reply:
The latter 👍